### PR TITLE
Added matrixMultiplication

### PR DESCRIPTION
**Pull request by grahamcracker1234**

### CODE REVIEW

**Review by lucalves — APPROVED** [review]

File: snippets/matrixMultiplication.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  for (let i = 0; i < aRows; i++) {
+    for (let j = 0; j < bColumns; j++) {
+      for (let k = 0; k < aColumns; k++) {
+        product[i][j] += a[i][k] * b[k][j];
+      }
+    }
+  }
+  return product;
+};
+```
```
Comment (Ilike2cum) [inline snippets/matrixMultiplication.md:25]: Mai

product;
};
```

```js
matrixMultiplication([[1, 2], [0, 1]![Uploading 132800452...]

Comment (Ilike2cum) [reply snippets/matrixMultiplication.md:25]: snippets/matrixMultiplication.me

Comment (Ilike2cum) [reply snippets/matrixMultiplication.md:25]: > Mai
> 
> Cc+++
>

File: snippets/matrixMultiplication.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  const [aRows, aColumns, bColumns] = [a.length, a[0].length, b[0].length];
+  const product = Array(aRows).fill().map(() => Array(bColumns).fill(0));
+  for (let i = 0; i < aRows; i++) {
+    for (let j = 0; j < bColumns; j++) {
+      for (let k = 0; k < aColumns; k++) {
+        product[i][j] += a[i][k] * b[k][j];
+      }
+    }
+  }
+  return product;
```
Comment (Ilike2cum) [inline snippets/matrixMultiplication.md:23]:
```suggestion
  return product;![image](https://user-images.githubusercontent.com/90334458/132800631-b28ee85e-4715-4b1c-95c5-a5b27250a8e1.jpeg)
```

Comment (Ilike2cum) [reply snippets/matrixMultiplication.md:23]: > ```suggestion
>   return product;image
> ```
> 

- Poor parenting

Comment (Ilike2cum) [reply snippets/matrixMultiplication.md:23]: product;
};
```

```js
matrixMultiplication([[1, 2], [0, 1]

**Review by Ilike2cum — APPROVED** [review]

**Comment by Ilike2cum** [comment]
```

```

File: snippets/matrixMultiplication.md (hunk 1 above)
Comment (Ilike2cum) [reply snippets/matrixMultiplication.md:25]: [[]()]()

**Comment by EdenEden11eden** [comment]
Hi

**Pull request closed without merging**